.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Angular on GitHub Actions
+
+on:
+  push:
+    branches: 
+      - github-actions
+
+jobs:
+  build:
+    name: Lint, Test, E2E
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
One more thing: we use 10.21 on our remote environment so we should use for the CI configuration too.